src/components/DeckMechanisms/index.js

```diff
@@ -468,6 +473,35 @@ export default class DeckMechanisms extends React.Component {
         break
       }
 
+      // Harvester of Souls
+      case 'N38': {
+        const id = arrayRandom(
+          cards
+            .filter(card => card.type === 'unit' && card.id !== 'T12')
```

Comment:
This has been done on master with https://github.com/KittySparkles/stormbound-kitty/commit/55e609e61d45d4d69414b2225ac52c627dde9703.